toltec/builder.py

```diff
@@ -316,13 +316,15 @@ def _build(self, recipe: Recipe, src_dir: str) -> None:
             )
 
         if host_deps:
-            opkg_conf_path = "$SYSROOT/etc/opkg/opkg.conf"
+            opkg_conf_path = "$SYSROOT_AARCH64/etc/opkg/opkg.conf" if recipe.arch == 'rmpp' else "$SYSROOT/etc/opkg/opkg.conf"
+            opkg_exec = "opkg-aarch64" if "rmpp" in recipe.arch else "opkg"
+
             pre_script.extend(
                 (
                     'echo -n "dest root /',
                     "arch all 100",
-                    "arch armv7-3.2 160",
-                    "src/gz entware https://bin.entware.net/armv7sf-k3.2",
+                    f"arch {'aarch64-3.10' if "rmpp" in recipe.arch else 'armv7-3.2'} 160",
```

Comment:
Since this would match `notrmpp` and we know we'll only be starting the architecture name with `rmpp`, the following would properly handle the edge case:
```suggestion
                    f"arch {'aarch64-3.10' if recipe.arch.startswith("rmpp") else 'armv7-3.2'} 160",
```

Reply:
Done!